.github/workflows/release.yml

```diff
@@ -4,94 +4,54 @@ on:
-
-    - name: Install cocoapods
-      run: gem install cocoapods
-
-    - name: Install xcodegen
-      run: brew install xcodegen
-
-    - name: Check version in Podspec
-      run: |
-        set -eo pipefail
-        echo Target version: ${{ github.event.inputs.tag }}
-        make check-version VERSION=${{ github.event.inputs.tag }}
-
-    - name: SPM integration test
-      if: ${{ github.event.inputs.action_tag == 'yes' }}
-      run: |
-        set -eo pipefail
-        echo SPM integration test starts:
-        make test-SPM-integration
-
-    - name: podspec file verification
-      if: ${{ github.event.inputs.action_tag == 'yes' }}
-      run: |
-        set -eo pipefail
-        echo podspec file verification starts:
-        make test-podspec
-        
-    - name: Build Artifacts
-      if: ${{ github.event.inputs.action_tag == 'yes' }}
-      run: |
-        make archive
-        make zip
-
-    - name: Create GH Release
-      id: create_release
-      uses: release-drafter/release-drafter@v5
-      if: ${{ github.event.inputs.action_tag == 'yes' }}
-      with:
-        name: v${{ github.event.inputs.tag }}
-        tag: ${{ github.event.inputs.tag }}
-        version: ${{ github.event.inputs.tag }}
-        publish: true
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Upload - AEPEdge
-      uses: actions/upload-release-asset@v1
-      if: ${{ github.event.inputs.action_tag == 'yes' }}
-      with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./build/AEPEdge.xcframework.zip
-        asset_name: AEPEdge-${{ github.event.inputs.tag }}.xcframework.zip
-        asset_content_type: application/zip
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Publish Pods - AEPEdge
-      if: ${{ github.event.inputs.release_AEPEdge == 'yes' }}
-      run: |
-        set -eo pipefail
-        pod trunk push AEPEdge.podspec --allow-warnings --synchronous
-        pod repo update
-      env:
-        COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
+      - name: Determine Extensions
+        id: set-extensions
+        run: |
+          FULL_RELEASE_EXTENSIONS=""
+          POD_RELEASE_EXTENSIONS=""
+
+          # Determine which extensions to include for release and publishing pods
+          if [ "${{ github.event.inputs.create-github-release }}" == "true" ]; then
+            FULL_RELEASE_EXTENSIONS="AEPEdge"
+          fi
+
+          if [ "${{ github.event.inputs.release-AEPEdge }}" == "true" ]; then
+            POD_RELEASE_EXTENSIONS="AEPEdge"
+          fi
+
+          # Output the variables to be used in the next job
+          echo "full_release_extensions=$FULL_RELEASE_EXTENSIONS" >> $GITHUB_OUTPUT
+          echo "pod_release_extensions=$POD_RELEASE_EXTENSIONS" >> $GITHUB_OUTPUT
+
+  release:
+    needs: set-extensions
+    uses: timkimadobe/aepsdk-commons/.github/workflows/ios-release.yml@gha-ios-1.0.0
```

Comment:
will we change to core/aepskd-commons?  Or will use your branch?

Reply:
Are you planning on updating to `adobe/aepsdk-commons` for this PR?